packages/behandling-opplaeringspenger/src/panelDefinisjoner/faktaPaneler/MedisinskVilkarFaktaPanelDef2.tsx

```diff
@@ -25,10 +24,9 @@ class MedisinskVilkarFaktaPanelDef2 extends FaktaPanelDef {
     behandlingType: behandling.type.kode,
   });
 
-  getOverstyrVisningAvKomponent = ({ fagsak, behandling }: { fagsak: Fagsak; behandling: Behandling }) => {
-    const erPleiepengesak = fagsak.sakstype.kode === fagsakYtelseType.PLEIEPENGER;
+  getOverstyrVisningAvKomponent = ({ behandling }: { fagsak: Fagsak; behandling: Behandling }) => {
     const søknadsfristErIkkeUnderVurdering = behandling.stegTilstand?.stegType?.kode !== 'VURDER_SØKNADSFRIST';
-    return erPleiepengesak && søknadsfristErIkkeUnderVurdering;
```

Comment:
Blir ikke noe stress for PILS ved fjerning av denne?

Reply:
dette er en egen for OLP